front/pages/api/v1/w/[wId]/spaces/[spaceId]/data_sources/[dsId]/documents/[documentId]/index.ts

```diff
@@ -529,7 +529,7 @@ async function handler(
             return apiError(req, res, {
               status_code: 403,
               api_error: {
-                type: "data_source_quota_error",
+                type: "workspace_quota_error",
```

Comment:
nit: as type to ensure there's not typo (it's something worth doing for other errors not touched in this PR)